requirements-dev.txt

```diff
@@ -18,4 +18,5 @@ pytest-cov==2.5.1
 six==1.11.0
 wrapt==1.10.11
 requests==2.20.1
-flask==1.0.2
\ No newline at end of file
+flask==1.0.2
+pyyaml==3.13
```

Comment:
Is pyyaml still needed? I can't find where it's used.

Reply:
ohh forgot to take it out. yeah dont need it anymore